src/stories.js

```diff
@@ -1,5 +1,6 @@
 import './shared/colors.css';
 import './shared/sizing.css';
+import './shared/buttons.css';
```

Comment:
I'm not 100%, but we might be able to drop these since we're doing the CSS loader config to make the variables available.

Reply:
when this was missing my buttons in storybook weren't working

Reply:
This was not working in Storybook because the loader changes were made in the rollup config, afaik

Reply:
Yup, so this is good.